rllib/agents/trainer.py

```diff
@@ -455,7 +460,7 @@ def train(self):
         result = None
         for _ in range(1 + MAX_WORKER_FAILURE_RETRIES):
             try:
-                result = Trainable.train(self)
+                result = super().train()
```

Comment:
Prefer Trainer.train() which makes it more clear where it is defined.

Reply:
True, adds clarity, I just don't like having to pass the `self`, then. Makes this error-prone (can be easily forgotten). I changed it back. Discuss at other time.